contrib/hack/setup-kubectl.sh

```diff
@@ -48,6 +48,6 @@ D_HOST=${D_HOST%:*}    # remove trailing port #
 # Setup our credentials
 NO_TTY=1 kubectl config set-credentials service-catalog-creds --username=admin --password=admin
 #NO_TTY=1 kubectl config set-cluster service-catalog-cluster --server=https://${D_HOST}:${PORT} --certificate-authority=/var/run/kubernetes-service-catalog/apiserver.crt
-NO_TTY=1 kubectl config set-cluster service-catalog-cluster --server=http://${D_HOST}:${PORT}
+NO_TTY=1 kubectl config set-cluster service-catalog-cluster --server=https://${D_HOST}:${PORT}
```

Comment:
we might actually not need this line at all because the server is basically guaranteed to change.